tajo-core/src/test/java/org/apache/tajo/engine/query/TestAlterTable.java

```diff
@@ -49,4 +50,18 @@ public final void testAlterTableAddNewColumn() throws Exception {
     assertColumnExists(createdNames.get(0),"cool");
   }
 
+  @Test
+  public final void testAlterTableSetProperty() throws Exception {
```

Comment:
I would like it more better If  you add more unit test cases such as, text delimiter, compression codec. :)